components/Item/Item.js

```diff
@@ -3,15 +3,96 @@ import React from 'react';
 const Item = ({ name }) => (
   <React.Fragment>
     <style jsx>{`
-      li {
-        border: 1px solid grey;
-        border-radius: 5px;
-        padding: 10px;
-        margin: 10px;
-        max-width: 500px;
+      s {
```

Comment:
`s` tag is not being used right?

Reply:
Yes, it is being used for striking the original price when discounted price is available.